pkg/eigenState/stakerShares/stakerShares.go

```diff
@@ -410,102 +449,120 @@ func parseLogOutputForSlashingWithdrawalQueuedEvent(outputDataStr string) (*slas
 }
 
 // handleM2QueuedWithdrawal handles the WithdrawalQueued event from the DelegationManager contract for M2.
-func (ss *StakerSharesModel) handleSlashingQueuedWithdrawal(log *storage.TransactionLog) ([]*StakerShareDeltas, error) {
+func (ss *StakerSharesModel) handleSlashingQueuedWithdrawal(log *storage.TransactionLog) ([]*StateDiff, error) {
 	outputData, err := parseLogOutputForSlashingWithdrawalQueuedEvent(log.OutputData)
 	if err != nil {
 		return nil, err
 	}
 
-	records := make([]*StakerShareDeltas, 0)
+	records := make([]*StateDiff, 0)
 
 	for i, strategy := range outputData.Withdrawal.Strategies {
 		shares, success := numbers.NewBig257().SetString(outputData.SharesToWithdraw[i].String(), 10)
 		if !success {
 			return nil, xerrors.Errorf("Failed to convert shares to big.Int: %s", outputData.SharesToWithdraw[i])
 		}
-		r := &StakerShareDeltas{
-			Staker:          outputData.Withdrawal.Staker,
-			Strategy:        strategy,
-			Shares:          shares.Mul(shares, big.NewInt(-1)).String(),
-			StrategyIndex:   uint64(i),
-			LogIndex:        log.LogIndex,
-			TransactionHash: log.TransactionHash,
-			BlockNumber:     log.BlockNumber,
+		r := &StateDiff{
+			Event: &StakerShareDeltas{
+				Staker:          outputData.Withdrawal.Staker,
+				Strategy:        strategy,
+				Shares:          shares.Mul(shares, big.NewInt(-1)).String(),
+				StrategyIndex:   uint64(i),
+				LogIndex:        log.LogIndex,
+				TransactionHash: log.TransactionHash,
+				BlockNumber:     log.BlockNumber,
+			},
 		}
 		records = append(records, r)
 	}
 	return records, nil
 }
 
-type AccumulatedStateChanges struct {
-	Changes []*AccumulatedStateChange
+type operatorSlashedOutputData struct {
+	Operator string `json:"operator"`
+	// OperatorSet struct {
```

Comment:
Remove if not needed

Reply:
https://github.com/Layr-Labs/sidecar/pull/68/commits/849bd0db77bd553611d6ccdf776de875882dc9c5